plugins/azure-devops/src/components/BuildTable/BuildTable.tsx

```diff
@@ -32,12 +32,14 @@ import {
   StatusWarning,
   Table,
   TableColumn,
+  EmptyState,
 } from '@backstage/core-components';
 
 import { AzurePipelinesIcon } from '../AzurePipelinesIcon';
 import { DateTime } from 'luxon';
 import React from 'react';
 import { getDurationFromDates } from '../../utils/getDurationFromDates';
+import { isAuthorizationError } from '../../utils';
```

Comment:
My general recommendation is to always lean into what Backstage offers instead of building custom patters like this which is why I recommend using `RequirePermission`. It also means less custom code we have to maintain ourselves.

Please change this in all the cases to `RequirePermission`, it might not be directly in this files. In this case it would probably be added here: https://github.com/backstage/backstage/blob/master/plugins/azure-devops/src/components/EntityPageAzurePipelines/EntityPageAzurePipelines.tsx